webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/hooks/RunNuCommandTest.scala

```diff
@@ -66,7 +66,7 @@ class RunNuCommandTest() extends Specification {
       val prog =
         for {
           p <- RunNuCommand.run(Cmd("env", Nil, Map("PATH" -> PATH, "foo" -> "bar")))
-          c <- p.await.timeout(500.millis).notOptional("oups, timed out")
+          c <- p.await.timeout(1500.millis).notOptional("oups, timed out")
```

Comment:
the failure is on "has only the 3 stdin, stdout, stderr file descriptors" test

Reply:
IT gives:

Reply:
indeed!